app/src/main/java/org/fossasia/openevent/general/ticket/TicketsFragment.kt

```diff
@@ -194,4 +200,22 @@ class TicketsFragment : Fragment() {
         val alert = builder.create()
         alert.show()
     }
+
+    private fun loadTickets() {
+        if (!isNetworkConnected(context) && ticketsViewModel.tickets.value.isNullOrEmpty())
+            showNoInternetScreen(true)
```

Comment:
Why you are showing no internet screen if tickets value is null or empty?

Reply:
Because if the tickets information is loaded and already retained in the ViewModel then I think we can just show that information, no need for no internet card

Reply:
@anhanh11001 This is wrong. It should be shown from the ViewModel. I am merging this, but create an issue to move all network logic to VM. Use BroadcastReceiver to make it reactive

Reply:
I will do it